core/src/main/resources/swagger/rawls.yaml

```diff
@@ -3759,6 +3759,9 @@ definitions:
       expression:
         type: string
         description: Expression that resolves to one or more entities matching the entity type in the method configuration.
+      readFromCache:
+        type: boolean
+        description: Whether or not to read from cache for this submission
```

Comment:
I would not use the word "read" because it can also be interpreted as past tense. How about "use" instead.